style.css

```diff
@@ -441,3 +462,239 @@ textarea {
   background-color: white;
   padding: 12px, 16px, 12px, 16px;
 }
+
+@media only screen and (max-width: 768px) {
+  .wide {
+    display: none;
+  }
+
+  .card1 {
+    background: url(img/Snapshoot-Portfolio-Mobile1.png);
+  }
+
+  .card2 {
+    background: url(img/Snapshoot-Portfolio-Mobile2.png);
+  }
+
+  .card3 {
+    background: url(img/Snapshoot-Portfolio-Mobile3.png);
+  }
+
+  .card4 {
+    background: url(img/Snapshoot-Portfolio-Mobile4.png);
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .mobile {
+    display: none;
+  }
+
+  main {
+    height: 916px;
+  }
+
+  nav {
+    height: 72px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    background-image: url(img/headline-bg-desktop.svg);
```

Comment:
- [ ] Please try to adjust your background image position in order to stick to the Figma template as much as possible. You can check the screenshot below to see :point_down: 

#### Figma template
![Screenshot from 2022-08-24 10-14-31](https://user-images.githubusercontent.com/6615394/186393663-4eaba585-f836-4d11-957a-bd2c3b299649.png)

#### Your design
![Screenshot from 2022-08-24 10-14-04](https://user-images.githubusercontent.com/6615394/186393673-3d3cd2d3-6cfe-435d-8015-f11b66a9f884.png)